### PR TITLE
fix tests

### PR DESCRIPTION
**Pull request by tiemevanveen**
Tests seemed broken, fixed it.

### CODE REVIEW

File: test/test-test.js [hunk 1]
```diff
@@ -33,6 +34,11 @@ function read (filepath) {
     return fs.readFileSync(path.join(__dirname, filepath), 'utf8');
 }
 
+function replaceHelpers (src) {
+    // Will replace the require for helpers so we can test the component
+    return src.replace("require('svg-react-loader/helpers')", "require('../helpers')");
+}
+
```
Comment (tiemevanveen) [inline test/test-test.js]: Without this tests we're failing. Not sure if this is the right approach though..

File: test/test-test.js [hunk 2]
```diff
@@ -67,26 +71,20 @@ describe('svg-react-loader', function () {
                     throw error;
                 }
 
-                var src = babel.transform(result, {
-                    presets: ['es2015', 'react']
-                }).code;
-                console.log(src);
-                fs.writeFileSync(__dirname + '/temp', src);
-                var el = react.createElement(require(__dirname + '/temp'));
-                var html = react.renderToStaticMarkup(el);
```
Comment (tiemevanveen) [inline test/test-test.js]: I got problems reading the file back in. It seems as if the previous file was read back in.. But I don't think you need to write to disk an read back in here. An `eval()` will also work fine.

File: test/test-test.js [hunk 3]
```diff
@@ -95,19 +93,11 @@ describe('svg-react-loader', function () {
...
+                var src = babel.transform(result, { presets: ['es2015', 'react'] }).code;
+                var el = react.createElement(eval(replaceHelpers(src)));
+                var html = reactDomServer.renderToStaticMarkup(el);
+                assert.equal('<svg><g><title>The Title</title><text x="20" y="20">Text</text></g></svg>', html);
```
Comment (tiemevanveen) [inline test/test-test.js]: added an assert for the expected output

File: test/test-test.js [hunk 4]
```diff
@@ -118,7 +108,16 @@ describe('svg-react-loader', function () {
...
-            callback: function (/*error, result*/) {
+            callback: function (error, result) {
+                if (error) {
+                  throw error;
+                }
+
+                var src = babel.transform(result, { presets: ['es2015', 'react'] }).code;
+                var el = react.createElement(eval(replaceHelpers(src)));
+                var html = reactDomServer.renderToStaticMarkup(el);
+                assert.equal('<svg width="auto" height="auto"><g><title>The Title</title><text x="20" y="20">Text</text></g></svg>', html);
```
Comment (tiemevanveen) [inline test/test-test.js]: added an assert for the expected output

**Comment by tera-sinube** [comment]
See my comments in issue #40

**Comment by tiemevanveen** [comment]
Check..

**Pull request closed without merging**